telluric/georaster.py

```diff
@@ -746,8 +746,13 @@ def _cleanup(self):
                 self._opened_files.pop(0)
 
         if self._filename is not None and self._temporary:
-            with contextlib.suppress(FileNotFoundError):
-                os.remove(self._filename)
+            with contextlib.suppress(OSError):
```

Comment:
this would hide issue in case of any concurrent acces to something you are deleting. is this what you want?

Reply:
Yes, this is the idea.